webpack/react_app/components/JobWizard/index.js

```diff
@@ -0,0 +1,32 @@
+import React from 'react';
+import { Title, Divider } from '@patternfly/react-core';
+import { translate as __ } from 'foremanReact/common/I18n';
+import PageLayout from 'foremanReact/routes/common/PageLayout/PageLayout';
+import { JobWizard } from './JobWizard';
+
+const JobWizardPage = () => {
```

Comment:
In other plugins I see folders `Components` and `Pages`.
I'm not advocating either, but I would like to set a standardized folder structure, so my question:
Do we want to keep Pages and components separate or not?

Reply:
As it's a page that has one component I like to keep them in the same folder. @laviro what do you think?

Reply:
I guess it's up to the plugin maintainers,
though I like the convention of `/webpack/${PAGE_NAME}` e.g: `webpack/JobWizard`

And then in routes registry point to this folder's index file

Reply:
@MariaAga yes, inside the main page folder I would use the components folder

Reply:
Moved to `webpack/JobWizard`, thanks @laviro!